products/tarantool.md

```diff
@@ -16,10 +16,15 @@ eolColumn: Support Status
 releaseDateColumn: false
 sortReleasesBy: 'releaseCycle'
 releases:
+  - releaseCycle: "2.10"
+    release: 2022-05-22
+    eol: false
+    latest: "2.10.0"
+
   - releaseCycle: "2.8"
-    release: 2021-12-22
+    release: 2022-04-25
```

Comment:
The release should be the date when the releaseCycle started, e.g. 2.8.0. You changed to the release date of the latest version, please revert.

Reply:
Fixed.